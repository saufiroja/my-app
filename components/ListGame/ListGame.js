<<<<<<< HEAD
import Image from "next/image";
import Link from "next/link";
import ResponsiveVideo from "../ResponsivePlayer";
=======
import Image from 'next/image';
import Link from 'next/link';
>>>>>>> main

export default function ListGame() {
  return (
    <section>
<<<<<<< HEAD
        <div className="container mx-auto">
          <h2 className="xl:text-8xl lg:text-7xl md:text-6xl sm:text-5xl font-bold font-body text-center">
            List Game
          </h2>
          <div className="flex w-9/12 mx-auto xl:mt-16 lg:mt-12 md:mt-8 sm:mt-8">
            <div className="w-4/12">
              <Link href="/login">
                <a>
                  <Image
                    className="cursor-pointer"
                    src="/images/game1.png"
                    alt="game1"
                    width={294}
                    height={217}
                  />
                </a>
              </Link>
            </div>
            <div className="w-4/12">
              <Link href="/login">
                <a>
                  <Image
                    className="cursor-pointer"
                    src="/images/game2.png"
                    alt="game2"
                    width={294}
                    height={217}
                  />
                </a>
              </Link>
            </div>
            <div className="w-4/12">
              <Link href="/login">
                <a>
                  <Image
                    className="cursor-pointer"
                    src="/images/game3.png"
                    alt="game3"
                    width={294}
                    height={217}
                  />
                </a>
              </Link>
            </div>
          </div>
        </div>

        {/* Section Video Tutorial */}
      {/* <section> */}
        <div className="container mt-14">
          <ResponsiveVideo
            url="https://www.youtube.com/watch?v=M8rbVzVf_z0"
          />
        </div>
      {/* </section> */}

        <div className="sticky">
          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320"><path fill="#5956E9" fillOpacity="1" d="M0,224L48,208C96,192,192,160,288,170.7C384,181,480,235,576,240C672,245,768,203,864,197.3C960,192,1056,224,1152,218.7C1248,213,1344,171,1392,149.3L1440,128L1440,320L1392,320C1344,320,1248,320,1152,320C1056,320,960,320,864,320C768,320,672,320,576,320C480,320,384,320,288,320C192,320,96,320,48,320L0,320Z"></path></svg>
        </div>
      </section>
=======
      <div className="container mx-auto">
        <h2 className="xl:text-8xl lg:text-7xl md:text-6xl sm:text-5xl font-bold font-body text-center">
          List Game
        </h2>
        <div className="flex w-9/12 mx-auto xl:mt-16 lg:mt-12 md:mt-8 sm:mt-8">
          <div className="w-4/12">
            <Link href="/login">
              <a>
                <Image
                  className="cursor-pointer"
                  src="/images/game1.png"
                  alt="game1"
                  width={294}
                  height={217}
                />
              </a>
            </Link>
          </div>
          <div className="w-4/12">
            <Link href="/login">
              <a>
                <Image
                  className="cursor-pointer"
                  src="/images/game2.png"
                  alt="game2"
                  width={294}
                  height={217}
                />
              </a>
            </Link>
          </div>
          <div className="w-4/12">
            <Link href="/login">
              <a>
                <Image
                  className="cursor-pointer"
                  src="/images/game3.png"
                  alt="game3"
                  width={294}
                  height={217}
                />
              </a>
            </Link>
          </div>
        </div>
      </div>

      <div className="sticky">
        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1440 320"><path fill="#5956E9" fillOpacity="1" d="M0,224L48,208C96,192,192,160,288,170.7C384,181,480,235,576,240C672,245,768,203,864,197.3C960,192,1056,224,1152,218.7C1248,213,1344,171,1392,149.3L1440,128L1440,320L1392,320C1344,320,1248,320,1152,320C1056,320,960,320,864,320C768,320,672,320,576,320C480,320,384,320,288,320C192,320,96,320,48,320L0,320Z" /></svg>
      </div>
    </section>
>>>>>>> main
  );
}