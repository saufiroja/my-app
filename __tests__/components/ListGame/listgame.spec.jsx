import React from 'react'
import { render, screen } from '@testing-library/react'
import ListGame from '../../../components/ListGame/ListGame'
import renderer from "react-test-renderer"

describe('ListGame', () => {
    it('renders a heading', () => {
        render(<ListGame />)

    const heading = screen.getByRole('heading', {
        name: "List Game",
        })

        expect(heading).toBeInTheDocument()
    })

    it('matches the snapshot', () => {
    const tree = renderer.create(<ListGame />).toJSON()

    expect(tree).toMatchSnapshot()
    })
})