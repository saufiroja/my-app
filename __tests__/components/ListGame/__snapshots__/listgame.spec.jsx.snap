// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListGame matches the snapshot 1`] = `
<section>
  <div
    className="container mx-auto"
  >
    <h2
      className="xl:text-8xl lg:text-7xl md:text-6xl sm:text-5xl font-bold font-body text-center"
    >
      List Game
    </h2>
    <div
      className="flex w-9/12 mx-auto xl:mt-16 lg:mt-12 md:mt-8 sm:mt-8"
    >
      <div
        className="w-4/12"
      >
        <a
          href="/login"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27294%27%20height=%27217%27/%3e"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="game1"
              className="cursor-pointer"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="game1"
                className="cursor-pointer"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2Fgame1.png&w=640&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fgame1.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fgame1.png&w=640&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </a>
      </div>
      <div
        className="w-4/12"
      >
        <a
          href="/login"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27294%27%20height=%27217%27/%3e"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="game2"
              className="cursor-pointer"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="game2"
                className="cursor-pointer"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2Fgame2.png&w=640&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fgame2.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fgame2.png&w=640&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </a>
      </div>
      <div
        className="w-4/12"
      >
        <a
          href="/login"
          onClick={[Function]}
          onMouseEnter={[Function]}
        >
          <span
            style={
              Object {
                "background": "none",
                "border": 0,
                "boxSizing": "border-box",
                "display": "inline-block",
                "height": "initial",
                "margin": 0,
                "maxWidth": "100%",
                "opacity": 1,
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "width": "initial",
              }
            }
          >
            <span
              style={
                Object {
                  "background": "none",
                  "border": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": "initial",
                  "margin": 0,
                  "maxWidth": "100%",
                  "opacity": 1,
                  "padding": 0,
                  "width": "initial",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                src="data:image/svg+xml,%3csvg%20xmlns=%27http://www.w3.org/2000/svg%27%20version=%271.1%27%20width=%27294%27%20height=%27217%27/%3e"
                style={
                  Object {
                    "background": "none",
                    "border": 0,
                    "display": "block",
                    "height": "initial",
                    "margin": 0,
                    "maxWidth": "100%",
                    "opacity": 1,
                    "padding": 0,
                    "width": "initial",
                  }
                }
              />
            </span>
            <img
              alt="game3"
              className="cursor-pointer"
              data-nimg="intrinsic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "width": 0,
                }
              }
            />
            <noscript>
              <img
                alt="game3"
                className="cursor-pointer"
                data-nimg="intrinsic"
                decoding="async"
                loading="lazy"
                src="/_next/image?url=%2Fimages%2Fgame3.png&w=640&q=75"
                srcSet="/_next/image?url=%2Fimages%2Fgame3.png&w=384&q=75 1x, /_next/image?url=%2Fimages%2Fgame3.png&w=640&q=75 2x"
                style={
                  Object {
                    "border": "none",
                    "bottom": 0,
                    "boxSizing": "border-box",
                    "display": "block",
                    "height": 0,
                    "left": 0,
                    "margin": "auto",
                    "maxHeight": "100%",
                    "maxWidth": "100%",
                    "minHeight": "100%",
                    "minWidth": "100%",
                    "objectFit": undefined,
                    "objectPosition": undefined,
                    "padding": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "width": 0,
                  }
                }
              />
            </noscript>
          </span>
        </a>
      </div>
    </div>
  </div>
  <div
    className="container mt-14"
  >
    <div
      className="relative pt-[56.25%]"
    >
      <div
        className="absolute top-0 left-0"
        style={
          Object {
            "height": "100%",
            "width": "100%",
          }
        }
      >
        <div
          style={
            Object {
              "display": undefined,
              "height": "100%",
              "width": "100%",
            }
          }
        >
          <div />
        </div>
      </div>
    </div>
  </div>
  <div
    className="sticky"
  >
    <svg
      viewBox="0 0 1440 320"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0,224L48,208C96,192,192,160,288,170.7C384,181,480,235,576,240C672,245,768,203,864,197.3C960,192,1056,224,1152,218.7C1248,213,1344,171,1392,149.3L1440,128L1440,320L1392,320C1344,320,1248,320,1152,320C1056,320,960,320,864,320C768,320,672,320,576,320C480,320,384,320,288,320C192,320,96,320,48,320L0,320Z"
        fill="#5956E9"
        fillOpacity="1"
      />
    </svg>
  </div>
</section>
`;
